### ISSUE TITLE
Adopt v2 Landscape

### ISSUE THREAD

**Issue by sftim** [kubernetes/website#44022]
**This is a Feature Request**

**What would you like to be added**
- Prepare to adopt https://cncf.landscape2.io/
- Actually adopt it

**Why is this needed**
https://cncf.landscape2.io/ will one day supersede https://landscape.cncf.io/guide

We use the CNCF landscape on pages such as https://kubernetes.io/training/

**Comments**
/area web-development

**Comment by kumarankit999** [comment]
How are we doing this @sftim

**Comment by Community-Programmer** [comment]
@sftim @kumarankit999 
Same Question how we are going to implement it? I think we can't embed it like we did in v1 as v2 uses Content Security Policy (CSP) settings

**Comment by tegioz** [comment]
Maybe this can help 🙂 

<https://l2.cncf.io/embed-setup>

<img width="1658" alt="training" src="https://github.com/kubernetes/website/assets/1213902/beda434c-9427-43da-82e0-f726e59e52b1">

**Comment by Community-Programmer** [comment]
@tegioz 
Thank you so much!👍
I am ready to work on this issue once it receives the triage/accepted label

**Comment by tegioz** [comment]
Hi!

We've migrated the CNCF landscape to v2 today. It'd be great if you could update the embed code when you have a chance. 

Thanks!

BTW <https://kubernetes.io/partners> seems to be affected as well.

**Comment by tegioz** [comment]
No worries!

In addition to the customization options available in the embed setup view, it's also possible to embed views using [iframe-resizer](https://github.com/davidjbradshaw/iframe-resizer). This feature can be enabled by adding `iframe-resizer=true` to the embed url ([codepen demo](https://codepen.io/cynthiasg/pen/WNmQjje), or you can see it in action in the [CNCF website](https://www.cncf.io/about/members/)).

/cc @nate-double-u

**Comment by nate-double-u** [comment]
Working on a fix, hope to have a PR up soon.

**Comment by Community-Programmer** [comment]
@nate-double-u 
 you are working on this?

**Comment by nate-double-u** [comment]
I am - but if you have something ready to go i'm happy to defer to you :)

**Comment by Community-Programmer** [comment]
@nate-double-u 

You can proceed; I haven't started yet. I'll unassign myself from the task :)

**Comment by nate-double-u** [comment]
@cynthia-sg, @tegioz said you might have a go at this since I'm running into a couple issues—thanks for helping 🙂

Some note as to where I got:
* I think [/layouts/shortcodes/cncf-landscape.html](https://github.com/kubernetes/website/blob/main/layouts/shortcodes/cncf-landscape.html) is key to this update.
* I noticed, I'm not sure there is a default, it seems like you always need to pass a `key` (what i this is replacing `category`). 
* I also found that if i pass `headers=false` i get an "invalid embed url" error, even though manually trying the same url (https://landscape.cncf.io/embed/embed.html?key=app-definition-and-development--application-definition-image-build&headers=false) in the browser works. This may be a red herring though, if everything is set up correctly it may work fine, i haven't had a chance to debug thoroughly yet.
